_data/sidebars/apim_3_x_sidebar.yml

```diff
@@ -178,7 +178,10 @@ entries:
                  url: /apim/3.x/apim_installguide_gateway_metrics.html
                - title: Enable OpenTracing
                  output: web
-                 url: /apim/3.x/apim_how_to_enable_opentracing_with_jaeger.html
+                 url: /apim/3.x/apim_enable_opentracing_with_jaeger.html
+               - title: OpenTracking in Docker
+                 output: web
+                 url: /apim/3.x/apim_opentracing_in_docker.html
```

Comment:
not sure this page should be in the configuration section
maybe more in "Developer guide"

I don't know

Reply:
I think you're right, but this is part of a bigger conversation. I'll leave it where it is until we've decided.